comp/snmptraps/bundle.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
It seems that comp/snmptraps/* is import modules from comp/netflow/* .

I see the intent, but I'm wondering if we should avoid that since the coupling between snmptraps and netflow is not natural.

I'm seeing 2 possible ways to avoid that:

1/ avoid converting into modules the dependencies like hostname/sender/testutil/sender for now

1b/ avoid converting into modules now, and wait for hostname/sender/testutil/sender/etc to be converted into components first.

2/ move shared modules to a shared package. 

---

I would favour 1/ since in general, during a refactor/componetization like those we should keep the scope as small when possible, by that I mean only change what's needed for componentization of the main package netflow/ndmtraps and leave other refactoring as separate PRs.